composer.json

```diff
@@ -19,7 +19,7 @@
     "require": {
         "php":             ">=5.3.2",
         "ext-curl":        "*",
-        "guzzle/guzzle":   "~3.7"
+        "guzzlehttp/guzzle": "~3.7 || ~4.0 || ~5.0 || ~6.0"
```

Comment:
No. I think support multiple versions would render this package more flexible.

See my comment: https://github.com/KnpLabs/php-github-api/issues/157#issuecomment-145344266

I know and understand that you want to use latests things (I like too), but as an open-source library, we have to satisfy multiple configuration IMHO.

Let contributors decide about it. If they want 6.x only, I'll update the PR.